README.md

```diff
@@ -61,9 +61,9 @@ The VirtualBox EFI implementation does not properly load the FileVault full disk
 ## Dependencies
 
 All the dependencies should be available through a package manager:  
-`bash` `coreutils` `gzip` `unzip` `wget` `xxd` `dmg2img`  `virtualbox`
+`bash` `coreutils` `gzip` `unzip` `wget` `xxd` `dmg2img`  `virtualbox` `ImageMagick`
```

Comment:
Should include `grep` as a dependency or remove `grep` from `compare_vb_screen_and_ref()`

Reply:
I may use something else compatible with coreutils.